modules/release-notes/pages/index.adoc

```diff
@@ -122,6 +122,28 @@ vertex function can also be used with vertex input parameters in the query-body
 
 == Fixed issues
 
+=== Fixed and Improved in 3.9.2-1
+
+==== Crashes and Deadlocks
+* Fixed a GPE crash caused by aborted query still being used by running thread.
+* Fixed crash issue after RESTPP auth refresh.
+* Fixed GSE crash due to the check failure when refreshing leader info after 60 minute timeout.
+* Fixed GPE crash and non-recovery due to Kafka retention to a higher offset.
+
+==== Functionality
+* Fixed data inconsistency while loading data and post-processing operations
+via write queries when running `data_consitency_check` with GPE and GSE.
+* Fixed command fail when user with only graph scope privilege runs a command that only requires graph scope privilege.
```

Comment:
Because graph scope privilege is common, this sounds like a pretty typical case. I'd want to take a closer look at the problem to make sure we are describing it correctly.

Reply:
I changed the one to something closer to what we had discussed over slack...

*Fixed `GRAPH` scope command failure due to missing privilege check in v3.9.2.